tla-io/src/main/scala/at/forsyte/apalache/tla/imp/OpApplTranslator.scala

```diff
@@ -61,12 +65,17 @@ class OpApplTranslator(
         case ASTConstants.UserDefinedOpKind =>
           translateUserOperator(node)
 
+        case ASTConstants.ConstantDeclKind =>
```

Comment:
Just to stay consistent with the comments above:

```suggestion
        // a higher-order operator declared as CONSTANT
        case ASTConstants.ConstantDeclKind =>
```